app/src/androidTest/java/com/swent/assos/EventDetailsTest.kt

```diff
@@ -141,12 +141,14 @@ class EventDetailsTest : SuperTest() {
     run {
       ComposeScreen.onComposeScreen<EventDetailsScreen>(composeTestRule) {
         step("Delete Event") {
-          composeTestRule.waitUntil(10000) { composeTestRule.onNodeWithTag("Button").isDisplayed() }
-          composeTestRule.onNodeWithTag("Button").performClick()
+          composeTestRule.waitUntil(10000) {
+            composeTestRule.onNodeWithTag("DeleteButton").isDisplayed()
+          }
+          composeTestRule.onNodeWithTag("DeleteButton").performClick()
```

Comment:
Watch out ! I put these test tags because the delete button changed. Please let them as `"Button"` because else it will not pass the CI